packages/terra-toggle-button/src/ToggleButton.module.scss

```
@@ -7,6 +7,10 @@
     display: inline-block;
     font-size: 0.8em;
     transform: rotate(0deg);
+
+    svg {
+      margin-top: 0.15em; // Style previously inherited from terra-button
```

Since original had the font-size of 0.8 being applied, move it down here and change from margin to position offset, for it to take correct effect:
```suggestion
      font-size: 0.8em;
      position: relative;
      top: -0.1em;
```